pkg/webhook/server/certificate/secret.go

```diff
@@ -12,33 +14,53 @@ import (
 	"github.com/qinqon/kube-admission-webhook/pkg/webhook/server/certificate/triple"
 )
 
-func tlsSecret(service types.NamespacedName, keyPair *triple.KeyPair) corev1.Secret {
+func updateTLSSecret(secret corev1.Secret, keyPair *triple.KeyPair) *corev1.Secret {
+	secret.Data = map[string][]byte{
+		corev1.TLSCertKey:       triple.EncodeCertPEM(keyPair.Cert),
+		corev1.TLSPrivateKeyKey: triple.EncodePrivateKeyPEM(keyPair.Key),
+	}
+	secret.Type = corev1.SecretTypeTLS
+	return &secret
+}
+
+func (m *Manager) newTLSSecret(serviceKey types.NamespacedName, keyPair *triple.KeyPair) (*corev1.Secret, error) {
+	service := corev1.Service{}
+	err := m.get(serviceKey, &service)
```

Comment:
please incorporate this line in the if statement:
`if err := m.get(serviceKey, &service); err != nil {`

Reply:
The rest of the project use different line, is no good to change that here.